README.md

```diff
@@ -25,22 +25,24 @@ consistency and integrity across the network without requiring generated code.
 
 This repository is the core of the `protovalidate` project. It contains:
 
-- [The API definition](proto/protovalidate/buf/validate/validate.proto): used to describe validation constraints
-- [Documentation](docs): how to apply `protovalidate` effectively
-- [Migration tooling](docs/migrate.md): incrementally migrate from `protoc-gen-validate`
-- [Examples](examples): example `.proto` files using `protovalidate`
-- [Conformance testing utilities](docs/conformance.md): for acceptance testing of `protovalidate` implementations
+- [The API definition][validate-proto]: used to describe validation constraints
```

Comment:
Instead of linking to the proto file, why don't we link to the BSR? https://buf.build/bufbuild/protovalidate/docs/main:buf.validate

Reply:
yes, i like that